### PR TITLE
Create Claimant Contact Information page

### PR DESCRIPTION
**Pull request by nickjg231**
## Original issue(s)
department-of-veterans-affairs/va-forms-system-core#0000

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link to the original github issue has been provided
- [ ] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: examples/burial-form/ClaimantContactInformation.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {AddressField, EmailField, Page, TextField} from '@department-of-veterans-affairs/va-forms-system-core';
+import {useFormikContext} from "formik";
+import PhoneField from "@department-of-veterans-affairs/va-forms-system-core/form-builder/PhoneField";
```
Comment (micahchiang) [inline examples/burial-form/ClaimantContactInformation.jsx:4]: You can import PhoneField like this: 

`import { PhoneField } from "@department-of-veterans-affairs/va-forms-system-core"`

File: examples/burial-form/ClaimantContactInformation.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export default function ClaimantContactInformation(props) {
+  const formikContext = useFormikContext();
+
+  formikContext?.values?.relationship?.isEntity = true;
+
+  return (
+    <>
+      <Page {...props} nextPage="/review-and-submit" prevPage="/benefits/plot-allowance">
+        {formikContext?.values?.relationship?.isEntity && (
```
Comment (micahchiang) [inline examples/burial-form/ClaimantContactInformation.jsx:14]: Does this need to be conditionally rendered? I could be missing something but it seems like `isEntity` is being assigned the value of `true` on line 9, so wouldn't this always just be rendered?

**Review by micahchiang — APPROVED** [review]
Looks good, please update the PhoneField import before merging. Thanks!

File: examples/burial-form/ClaimantContactInformation.jsx (hunk 2 above)
Comment (nickjg231) [reply examples/burial-form/ClaimantContactInformation.jsx:17]: Those two fields only can render if `isEntity = true`, however the code setting `isEntity = true` was just for testing. I have removed that line after making sure it works for both cases

File: examples/burial-form/ClaimantContactInformation.jsx (hunk 1 above)
Comment (nickjg231) [reply examples/burial-form/ClaimantContactInformation.jsx]: Updated!

**Pull request merged**